lib/ansible/modules/apt_key.py

```diff
@@ -406,6 +406,10 @@ def main():
         mutually_exclusive=(('data', 'file', 'keyserver', 'url'),),
     )
 
+    global lang_env
+    locale = get_best_parsable_locale(module)
+    lang_env = dict(LANG=locale, LC_ALL=locale, LC_MESSAGES=locale)
```

Comment:
Instead of making `lang_env` global, make it a function that caches its result. Something like:

```python
def lang_env():
    try:
        return lang_env.result
    except AttributeError:
        pass

    locale = get_best_parsable_locale(module)
    lang_env.result = dict(LANG=locale, LC_ALL=locale, LC_MESSAGES=locale)

    return lang_env.result
```

Reply:
i was tempted to do that, but wanted to minimize the change.

Reply:
This change should fix the unit test failures.

Reply:
done